cell_classification/segmentation_data_prep_test.py

```diff
@@ -142,15 +162,68 @@ def test_check_input():
 
         # check if the normalization_dict is calculated in check_input when
         # data_folders but no normalization_dict_path is given to init
-        data_prep = prep_object(
-            data_folders=data_folders,
-            conversion_matrix_path=conversion_matrix_path,
-            tf_record_path=os.path.join(temp_dir, "tf_record_path"),
-            cell_table_path=cell_table_path,
-        )
+        data_prep.data_folders = data_folders
+        data_prep.normalization_dict_path = None
         data_prep.check_input()
         assert norm_dict == data_prep.normalization_dict
 
+        # check if ValueError is raised if selected_markers in conversion_matrix
```

Comment:
We should add a direct check to make sure that selected_markers is present in one of the example images. 

If the user says "CD455" but means CD45, makes more sense to raise an error directly saying "CD455" not found in image data, rather than saying CD45 not found in conversion matrix